rust-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "1.65"
```

Comment:
I'm not sure whether that's a good idea, honestly. It'll trigger downloading a toolchain, usually unnecessarily, for all developers. We should probably just have an MSRV policy and test for it in CI.

As for all crates in the Whisperfish group, I'd like to vote for 1.62, but I'm not going to make that a hard requirement if any good arguments exist.